ivy/data_classes/container/base.py

```diff
@@ -2845,7 +2845,7 @@ def cont_overwrite_at_key_chains(
                 )
             else:
                 return_dict[k] = v
-        return ivy.Container(return_dict, **self._config)
+        return return_dict
```

Comment:
let's use [this](https://github.com/unifyai/ivy/pull/18164#discussion_r1250803939) return statement instead

Reply:
But then, in the case of `inplace=True` it won't return `self` , it will return a copy. Better alternative: ` return return_dict if inplace else ivy.Container(return_dict, **self._config)`

Reply:
the whole `**self._config` logic applies only to `inplace=False` I guess

Reply:
@saeedashrraf what do you think?

Reply:
I don't think the config logic would apply to `inplace=False` either, given that `self.cont_copy` also creates a container with the same `config`. Probably we can use `return return_dict` as you mentioned